test/core.jl

```diff
@@ -3306,6 +3306,7 @@ end
 @test Tuple{Int} === Tuple{Int, Vararg{Integer, 0}}
 
 # issue #12003
+using Dates
```

Comment:
same here; I think it'd be generally preferable to not have Core/Base tests depend on `stdlib` modules, but I guess this is ok for now.

Reply:
How about the `Test` stdlib module? :trollface: